README.md

```diff
@@ -125,11 +125,11 @@ This project is set up to use [jest](https://facebook.github.io/jest/) for tests
 
 ## End-to-End Tests
 
-This repository extends the end-to-end (E2E) tests found in the Mobile Gutenberg (MG) [project repository](https://github.com/WordPress/gutenberg). The majority of E2E tests should reside in the MG repository. However, experimental tests or non-core-related tests should be placed in the `gutenberg-mobile` repository.
+This repository extends the end-to-end (E2E) tests found in the [Gutenberg project repository](https://github.com/WordPress/gutenberg). The majority of E2E tests should reside in the `gutenberg` repository. However, experimental tests or non-core-related tests should be placed in the `gutenberg-mobile` repository.
 
-Reviewing the MG E2E test [documentation](https://github.com/WordPress/gutenberg/tree/trunk/packages/react-native-editor/__device-tests__#readme) is the best approach for understanding the test environment and how to set up your computer to run tests. Much of the information and approaches outlined there also apply to the tests found in this repository. The only difference being that this repository includes its own matching npm scripts to run the test, e.g. `npm run test:e2e:ios:local`.
+Reviewing the [Mobile Gutenberg E2E test documentation](https://github.com/WordPress/gutenberg/tree/trunk/packages/react-native-editor/__device-tests__#readme) is the best approach for understanding the test environment and how to set up your computer to run tests. Much of the information and approaches outlined there also apply to the tests found in this repository. The only difference being that this repository includes its own matching npm scripts to run the test, e.g. `npm run test:e2e:ios:local`.
```

Comment:
Not a blocker, but I noticed we reference the project differently across this file. The title says `Mobile Gutenberg` and in other areas ([example](https://github.com/wordpress-mobile/gutenberg-mobile/blob/4586443f1b2bbc3483923b457c637208a269afe7/README.md#internationalization-i18n)) it says `Gutenberg Mobile`. It would be great if we use the same name for consistency.

Reply:
I agree there is opportunity to improve consistency and clarity. 

For this line in particular, this sentence intended to reference `gutenberg`, not `gutenberg-mobile`. So, `MG` should likely be replaced with `gutenberg` to match earlier proposed changes on line ~128. I'll update the PR to address this.